src/styles/StyleUtils.js

```diff
@@ -448,6 +448,29 @@ function getPaddingLeft(paddingLeft) {
     };
 }
 
+/**
+ * Generate a style for the ImageCropView, based on its container size.
+ * @param {Number} containerSize
+ * @returns {Object}
+ */
+function getImageCropViewStyle(containerSize) {
+    return {
+        height: containerSize,
+        width: containerSize,
+    };
+}
```

Comment:
Instead of this we can use `getWidthAndHeightStyle` function. So this can be removed.